src/main/java/com/dubture/jenkins/digitalocean/ComputerLauncher.java

```diff
@@ -128,8 +113,16 @@ public void onClosed(Channel channel, IOException cause) {
 
             successful = true;
         } catch (Exception e) {
+            try {
+                Jenkins.getInstance().removeNode(computer.getNode());
+            } catch (Exception ee) {
+                ee.printStackTrace(logger);
+            }
             e.printStackTrace(logger);
         } finally {
+            Date endDate = new Date();
+            logger.println("Done setting up at: " + getUtcDate(endDate));
```

Comment:
This is being printed out into Node's log in Jenkins web UI, not in the log file, so the log level doen't matter in there. Also, if you didn't notice, all code in `ComputerLauncher.java` does this, so I'm not sure why you are having issue just with this single case.